frontend/src/container/Sidebar.jsx

```diff
@@ -171,6 +171,11 @@ export class Sidebar extends React.Component {
           </button>
         </div>
 
+        <div
+          id='customToolboxHeaderBtn'
+          className='header__menu__rightside__specificBtn'
+        />
```

Comment:
the #customToolboxHeaderBtn also is in file frontend/src/container/Header.jsx

Isn't it twice in the page then?

Reply:
No the header only appears when the user is not logged and the sidebar only when they are.

Reply:
understood
seems good to me then